integration-tests/tests/aws/post-questions/post-questions-happy.test.ts

```diff
@@ -0,0 +1,126 @@
+import { stackOutputs } from "../resources/cloudformation-helper";
+import { clearItems, populateTable } from "../resources/dynamodb-helper";
+import { executeStepFunction } from "../resources/stepfunction-helper";
+
+describe("HMRC KBV Post Question", () => {
```

Comment:
for consistency, can we follow the convention in the other state machines i.e. `post-questions-happy`